static/src/javascripts/projects/common/modules/media/videojs-plugins/fullscreener.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import bean from 'bean';
+import bonzo from 'bonzo';
+
+/**
+    videojs plugins can't use arrow functions
+    as 'this' needs to be available as it 
+    represents an instance of the videojs player
+**/
+
+// eslint-disable-next-line func-style
+function fullscreener(): void {
+    const player = this;
+    const clickbox = bonzo.create(
+        '<div class="vjs-fullscreen-clickbox"></div>'
+    )[0];
+    const events = {
+        click(e: bean): void {
+            if (player.paused()) {
+                player.play();
+            } else {
+                player.pause();
+            }
+            e.stop();
+        },
+        dblclick(e: bean): void {
+            e.stop();
+            if (player.isFullscreen()) {
+                player.exitFullscreen();
+            } else {
+                player.requestFullscreen();
+            }
+        },
+    };
+
+    bonzo(clickbox).appendTo(player.contentEl());
+
+    bean.on(clickbox, 'click', events.click);
+    bean.on(clickbox, 'dblclick', events.dblclick);
+
+    player.on('fullscreenchange', (): void => {
+        if (player.isFullscreen()) {
+            player.trigger('player:fullscreen');
```

Comment:
Since this handler is an arrow function, `player` and `this` should be equal (confirm with `console.log(player === this)`). If so, we can dispense with the `player` assignment altogether